CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ set(CMAKE_FIND_PACKAGE_PREFER_CONFIG ON)
 set(minBoostVersion 1.65)
 set(minQtVersion 5.14)
 set(minOpenSSLVersion 1.1.1)
-set(minLibtorrentVersion 1.2.13)
+set(minLibtorrentVersion 1.2.14)
```

Comment:
@FranciscoPombal
Could we provide restrictions for v2.0 branch too? E.g. if `libtorrent >= 1.2.14` is found check whether its version belongs to v2.0 branch and invoke findPackage with 2.0.4?

Reply:
Actually I would drop support for libtorrent-1.2 in qBittorrent v4.4+.
@sledgehammer999, what do you think?

Reply:
@glassez I would much prefer deferring that to another PR. Is that OK?

Reply:
>@glassez I would much prefer deferring that to another PR. Is that OK?

Yes.